doc/api/http.markdown

```diff
@@ -43,12 +43,6 @@ list like the following:
       'Host', 'mysite.com',
       'accepT', '*/*' ]
 
-## http.METHODS
-
-* {Array}
-
-A list of the HTTP methods that are supported by the parser.
-
```

Comment:
Can we move to a model where we just support *any* method? I argued with ry back in the day about this, but people at the IETF add HTTP methods for crazy new protocols all the time. Ryan hated this and so he intentionally limited the number of methods supported but there's no real technical reason for it.